### DISCUSSION TITLE
Weird behavior with --ffmpeg-copyts

### DISCUSSION THREAD

**Discussion by campones** [streamlink/streamlink#5121] [Q&A]
Hello

I have been using this for like a month, worked great overall.  Today for some reason, ffmpeg couldn't detect the codec parameters and suggest me to use some more parameters to help get the video track details. 
I could get rid of the issue (apparently) by removing --ffmpeg-copyts in the streamlink command

/usr/local/bin/streamlink --ffmpeg-ffmpeg "/usr/local/bin/ffmpeg5" --ffmpeg-audio-transcode "aac"  --ffmpeg-copyts --ffmpeg-fout "flv" "https://www.france.tv/france-2/direct.html" 540p

/usr/local/bin/streamlink --ffmpeg-ffmpeg "/usr/local/bin/ffmpeg5" --ffmpeg-audio-transcode "aac" --ffmpeg-fout "flv" "https://www.france.tv/france-2/direct.html" 540p

[flv @ 0x55ffcc67ddc0] Could not find codec parameters for stream 0 (Video: h264, none): unspecified size
Consider increasing the value for the 'analyzeduration' and 'probesize' options
Input #0, flv, from 'pipe:0':
  Metadata:
    encoder         : Lavf59.27.100
  Duration: 00:00:00.00, start: 0.000000, bitrate: N/A
    Stream #0:0: Video: h264, none, 25 fps, 1k tbr, 1k tbn
    Stream #0:1: Audio: aac (LC), 48000 Hz, stereo, fltp, 128 kb/s
[flv @ 0x55ffcc68bf80] dimensions not set
Could not write header for output file #0 (incorrect codec parameters ?): Invalid argument

So my idea is:  those options could be added to streamlink? like --ffmpeg-analyzeduration 100M -ffmpeg-probesize 100M
Because I think I m pretty limited in term of options I can add while using ffmpeg inside streamlink command.

**Comment by bastimeyer** [comment]
This is once again a post where your intention of restreaming content with the help of Streamlink is very clear. We don't support any of that. You'll have to figure this out on your own.

And in regards to adding more FFmpeg parameters to Streamlink, it doesn't make sense adding parameters for each option supported by libavformat. A more generic option would need to be added.

**Comment by campones** [comment]
still it' s a bit weird that while not doing any changes on my side, this stopped working?  
Only by removing --ffmpeg-copyts I can actually retrieve properly the video codec and therefore the dimension. 

They must have changed something or it's just a temporary bug.  still working as before for france 5 ?
